lib/sentry_dart_plugin.dart

```diff
@@ -149,6 +149,11 @@ class SentryDartPlugin {
 
     _addWait(releaseDartFilesParams);
 
+    if (_configuration.urlPrefix != null) {
+      releaseDartFilesParams.add('--url-prefix');
+      releaseDartFilesParams.add(_configuration.urlPrefix!);
```

Comment:
Any chance you can also add a test for that?

Reply:
done :)